app/renderer/main.html

```diff
@@ -41,4 +41,13 @@
 </body>
 <script src="js/main.js"></script>
 
+<!-- To trigger electron.reload on changes in renderer from gulp -->
+<script>
+  window.addEventListener('load', () => {
+    const isDev = require('electron-is-dev');
+    if (isDev) {
+      require('electron-connect').client.create();
```

Comment:
@cPhost, @abhigyank this also called when you run `npm start` which we don't want :/

Reply:
That is weird.

Reply:
I don't think this will be the case in packaged or distributed version. Since, ```npm start``` is basically just ```electron app```, it is possibly treated as development.